com.engine.Acestream.json

```diff
@@ -0,0 +1,142 @@
+{
+    "app-id": "com.engine.Acestream",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "acestream",
+    "tags": [
+        "proprietary"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=/tmp"
+    ],
+    "modules": [
+            "libs/intltool/intltool-0.51.json",
+            "libs/gtk2/gtk2.json",
+            "libs/python2.7/python-2.7.json",
+            "libs/dbus-glib/dbus-glib-0.110.json",
```

Comment:
Use the ones from shared-modules please.

Reply:
gtk2 - function duplicate (with flatpak-builder an error is thrown)
    "../intltool/intltool-0.51.json", - line 35

Reply:
everything is done, except for gtk2 (because of the module)

Reply:
done (including gtk2)